graphql/resolvers/directives.js

```diff
@@ -0,0 +1,62 @@
+import fs from 'fs';
+import path from 'path';
+import { forEachField } from 'graphql-tools';
+import { getArgumentValues } from 'graphql/execution/values';
+import jwt from 'jsonwebtoken';
+import debug from 'debug';
+
+import { AuthorizationError } from '../errors';
+import { asyncErrorHandler } from '../../utils';
+
+const log = debug('fcc:resolvers:directives');
+const {
+  NODE_ENV,
+  STAGING_JWT_CERT: stagingCert,
```

Comment:
How about we just have `JWT_CERT`, and our deploy script figures out which variables are used, like we do for `MONGODB_URL` and `GRAPHQL_ENDPOINT_URL`? If we do that we'll need to edit `scripts/deploy.sh`

In any case you'll still need to add the env variables to `serverless.yml`.